### PR TITLE
add new reporters section in the readme

### PR DESCRIPTION
**Pull request by MK-61**
Fixes / updates #898

### CODE REVIEW

**Comment by nfcampos** [comment]
that screenshot is for npm's verbose mode not ava´s. you'd have to run `npm test -- --verbose` to get ava's verbose mode, without the extra `--` you're passing the verbose flag to npm itself

**Comment by MK-61** [comment]
I am very sorry for that. AVA is new to me, and I am new to new technologies.  As I understand correctly - old screenshot was sufficient, so I put it instead.

**Comment by nfcampos** [comment]
No need to apologise :) passing flags to `npm run` is rather confusing

File: readme.md [hunk 1]
```diff
@@ -169,6 +167,37 @@ Directories are recursed, with all `*.js` files being treated as test files. Dir
...
+
+![](media/screenshot-mini-reporter.gif)
+
+### Verbose reporter
+
+The verbose reporter is always used in CI environments unless [`--tap`](#optional-tap-output) is specified.
+
+<img src="media/screenshot.png" width="150">
+
+### Optional TAP output
```
Comment (sindresorhus) [inline readme.md]: > Tap reporter

File: readme.md [hunk 2]
```diff
@@ -169,6 +167,37 @@ Directories are recursed, with all `*.js` files being treated as test files. Dir
 
 When using `npm test`, you can pass positional arguments directly `npm test test2.js`, but flags needs to be passed like `npm test -- --verbose`.
 
+## Reporters
+
+### Mini-reporter
+
```
Comment (sindresorhus) [inline readme.md]: Mention that it's the default reporter.

File: readme.md [hunk 3]
```diff
@@ -169,6 +167,37 @@ Directories are recursed, with all `*.js` files being treated as test files. Dir
...
+![](media/screenshot-mini-reporter.gif)
+
+### Verbose reporter
+
```
Comment (sindresorhus) [inline readme.md]: Mention that it can be selected with the `--verbose` flag.

File: readme.md [hunk 4]
```diff
@@ -169,6 +167,37 @@ Directories are recursed, with all `*.js` files being treated as test files. Dir
 
 When using `npm test`, you can pass positional arguments directly `npm test test2.js`, but flags needs to be passed like `npm test -- --verbose`.
 
+## Reporters
```
Comment (sindresorhus) [inline readme.md]: Needs to be added to the Table of Contents.

File: readme.md (hunk 1 above)
Comment (sindresorhus) [inline readme.md]: Also need to update the link in "Why AVA?" then.

Comment (jamestalmage) [inline readme.md]: All caps (`TAP`) is the preferred representation based on the website http://testanything.org

**Comment by MK-61** [comment]
'Optional TAP output' >> 'TAP reporter',
links in 'Why AVA?' updated, 
Mini-reporter mentioned that it's the default reporter,
Verbose reporter mentioned that it can be selected with the `--verbose` flag,
Reporters section added to the Table of Contents.

**Comment by sindresorhus** [comment]
Looks good. Thank you @MK-61 :)

**Pull request merged**